mesonbuild/interpreter.py

```diff
@@ -1214,6 +1213,87 @@ def get_cross_property_method(self, args, kwargs):
                 return args[1]
             raise InterpreterException('Unknown cross property: %s.' % propname)
 
+pch_kwargs = set(['c_pch', 'cpp_pch'])
+
+lang_arg_kwargs = set(['c_args',
+                       'cpp_args',
+                       'java_args',
+                       'rust_args',
+                       'objc_args',
+                       'objcpp_args',
+                       'fortran_args',
+                       'vala_args',
+                       'd_args',
+                   ])
+
+vala_kwargs = set(['vala_gir', 'vala_header'])
+rust_kwargs = set(['rust_crate_type',
+               ])
+cs_kwargs = set(['resources'])
+
+buildtarget_kwargs = set(['sources',
+                          'link_with',
+                          'link_whole',
+                          'link_args',
+                          'link_depends',
+                          'include_directories',
+                          'dependencies',
+                          'gui_app',
+                          'extra_files',
+                          'install',
+                          'install_rpath',
+                          'install_dir',
+                          'objects',
+                          'native',
+                          'name_suffix',
+                          'name_prefix',
+                          'build_by_default',
+                          'vs_module_defs',
+                          'pic',
+                          'override_options',
+                      ])
```

Comment:
__[flake8]__
* [E124] closing bracket does not match visual indentation

<!-- Comment by [SideCI](https://sideci.com) -->